cmd/thanos/compact.go

```diff
@@ -456,10 +457,12 @@ func runCompact(
 		global.Register(r, false, ins)
 
 		// Configure Request Logging for HTTP calls.
-		opts := []logging.Option{logging.WithDecider(func() logging.Decision {
-			return logging.NoLogCall
-		})}
-		logMiddleware := logging.NewHTTPServerMiddleware(logger, opts...)
+		logOpts, err := logging.NewHTTPLoggingOption(reqLogYAML)
+		if err != nil {
+			level.Error(logger).Log("msg", "config YAML for request logging not recognized", "error", err)
```

Comment:
This should be hard error, otherwise how user can find out if request logging was configured well?

Reply:
return error (:

Reply:
Done